.github/CODEOWNERS

```diff
@@ -13,4 +13,5 @@
 /deploy/scripts/src/cri-orange @alphagov/digital-identity-performance-test @alphagov/di-ipv-orange-cri-maintainers
 /deploy/scripts/src/ipv-core @alphagov/digital-identity-performance-test @alphagov/digital-identity-ipv
 /deploy/scripts/src/mobile @alphagov/digital-identity-performance-test @alphagov/di-doc-checking-narwhal-codeowners
-/deploy/scripts/src/txma @alphagov/digital-identity-performance-test @alphagov/digital-identity-txma
\ No newline at end of file
+/deploy/scripts/src/txma @alphagov/digital-identity-performance-test @alphagov/digital-identity-txma
+/deploy/scripts/src/spot @alphagov/digital-identity-performance-test @alphagov/di-ipv-spot-team
```

Comment:
Switch these two so it's alphabetical please 😄

Reply:
Done in the next commit